core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts

```diff
@@ -546,12 +546,6 @@ export class EnvLogsSectionComponent implements OnInit {
 			this.assetsNameArray.push(response);
 			let assets=_(response.data.assets).map('asset_type').uniq().value();
 
-			// TODO: Consider hoisting to member or configuration
```

Comment:
Can you clarify why this is removed?

Reply:
Without this, all asset types will show in the Asset Type filter for logs (similarly we have for metrics). Only some asset types have logs and metrics.

Reply:
Though I have removed the above filter, there is a list of supported asset types provided in env.oss file, which is listed below. So asset names filter will be shown for all those asset types. If below all Asset types don't support for asset name filter, then I can re-add only for those we require it. 
assetTypeList: [
    "lambda",
    "apigateway",
    "cloudfront",
    "s3",
    "dynamodb",
    "cloudwatch_event",
    "dynamodb_stream",
    "iam_role",
    "sqs",
    "kinesis_stream",
    "apigee_proxy"
  ]

Reply:
After discussion with Prakash, removed whitelist for asset types is re-added.